docs/data/pages.ts

```diff
@@ -125,6 +125,30 @@ const pages: MuiPage[] = [
       },
     ],
   },
+  {
+    pathname: '/toolpad/examples',
```

Comment:
Based on https://github.com/mui/mui-toolpad/issues/2052#issue-1719145680, we should add "Examples" as a sub-section to the "Getting started" section

Reply:
I assumed we'll do it under the restructuring PR. 
Also since we don't support 3 levels of hierarchy in the docs, does that mean putting "Examples" under "Getting started" section would mean: The 3 examples would come on the right navigation?

Reply:
Not sure what you mean but I've added the change in the latest commit to this PR

Reply:
Looks fine, thanks.